AsyncDisplayKit/Details/ASDataController.h

```diff
@@ -95,7 +112,7 @@ extern NSString * const ASCollectionInvalidUpdateException;
 /**
  Called for insertion of sections.
  */
-- (void)dataController:(ASDataController *)dataController didInsertSections:(NSArray<NSArray<ASCellNode *> *> *)sections atIndexSet:(NSIndexSet *)indexSet withAnimationOptions:(ASDataControllerAnimationOptions)animationOptions;
+- (void)dataController:(ASDataController *)dataController didInsertSectionsAtIndexSet:(NSIndexSet *)indexSet withAnimationOptions:(ASDataControllerAnimationOptions)animationOptions;
```

Comment:
Yes! Very happy to see this argument removed. Let's phrase it `didInsertSections:(NSIndexSet *)sectionIndexes` since it's clear that we're passing an index set.